data/two-squares-nurbs-rot.mesh

```diff
@@ -0,0 +1,61 @@
+MFEM NURBS mesh v1.0
+
+# 2D Two squares mesh consisting of to patches 
```

Comment:
This description has typos and should be rewritten more carefully.

Reply:
Fixed.